### PR TITLE
US08_Saja

### PR DESCRIPTION
**Pull request by sajawii**
please review US08

### CODE REVIEW

File: src/test/java/net/meetsky/runners/AlulaRunner.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
                 "json:target/cucumber.json"
         },
         publish = true,
-        tags = "",
+        tags = "@Saja",
```
Comment (vsimpryvit) [inline src/test/java/net/meetsky/runners/AlulaRunner.java:20]: Saja, could you remove this tag from the runner to avoid this minor conflict

**Review by vsimpryvit — CHANGES_REQUESTED** [review]
Hey Saja. I left a comment on the line for the runner class. You left your tag on, that might cause a minor conflict that we could avoid. Just remove it and send your PR again.
Other than that great job! Thanks:)

File: src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java [hunk 2]
```diff
@@ -280,4 +280,30 @@ public void verify_the_app_displays_the_expected_result_option() {
         Assert.assertEquals(ConfigurationReader.getProperty("searchValue"), searchPage.displayedResult.getText());
     }
 
+
+    //Saja US
+    FilesPage filePage = new FilesPage();
+
+    @When("user click \"new folder”")
```
Comment (Sayora1) [inline src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java:287]: Hey Saja, this method should be parameterized. Make sure your feature file is correct.

File: src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java [hunk 3]
```diff
@@ -280,4 +280,30 @@ public void verify_the_app_displays_the_expected_result_option() {
         Assert.assertEquals(ConfigurationReader.getProperty("searchValue"), searchPage.displayedResult.getText());
     }
 
+
+    //Saja US
+    FilesPage filePage = new FilesPage();
```
Comment (Sayora1) [inline src/test/java/net/meetsky/step_definitions/Meetsky_StepDefinitions.java:285]: Can you put this line on top of others codes.

**Pull request merged**